test/common/wpt.js

```diff
@@ -752,7 +752,7 @@ class WPTRunner {
 
   getTestTitle(filename) {
     const spec = this.specMap.get(filename);
-    return spec.meta?.title || filename;
+    return spec.meta?.title || filename.split('.')[0];
```

Comment:
Is there any chance `filename` can be undefined? 
```suggestion
    return spec.meta?.title || filename?.split('.')[0];
```

Reply:
No.